src/components/enterprise-page/data/hooks.js

```diff
@@ -6,11 +6,13 @@ import { fetchEnterpriseCustomerConfig } from './service';
 
 export const defaultBorderColor = '#007D88';
 export const defaultBackgroundColor = '#D7E3FC';
+export const defaultSearchBackgroundColor = '#1a337b';
```

Comment:
Note: These color configurations for the search banner will go away with https://github.com/edx/frontend-app-learner-portal-enterprise/pull/36.